### ISSUE TITLE
support debhelper 10

### ISSUE THREAD

**Issue by bkw777** [bkw777/mainline#74]
debian wants to depreciate debian/compat level 9, but compat level 10 fails when trying to build the deb package.
"make" and "make install" work fine, only "make deb" or "make release_deb" fails.

with debian/compat = 9:  
builds fine.

with debian/compat = 10:  
two vala files are ignored, for no reason I can fathom

    cc1: fatal error: /build/mainline-1.0.14/src/Utility/TeeJee.ProcessHelper.vala.c: No such file or directory
    compilation terminated.
    cc1: fatal error: /build/mainline-1.0.14/src/Utility/TeeJee.System.vala.c: No such file or directory
    compilation terminated.

These two files are listed as sources in Makefile the same way as the other files which are built normally.  
Relevant excerpts from Makefile:

    common_vala_files := src/Common/*.vala src/Utility/*.vala
    ...
    valac ... $(common_vala_files) $(gui_vala_files) -o $(@)
    ...
    valac ... $(common_vala_files) $(tui_vala_files) -o $(@)

The .vala files that generate the missing .vala.c files are included in the globbing pattern in common_vala_files, and are included on the valac command line for both the commandline and -gtk targets.  

The full build log does not show any earlier error where the .vala files were found and tried but failed to generate their .vala.c file. The two .vala.c files are missing because the two .vala files were simply ignored for some reason.

And of course, the app has already been building just fine for years, so that would seem to prove that the Makefile is correct.

Only when setting debian/compat to 10, it fails like this.

**Comment by Uldiniad** [comment]
I'm having the same issue on Ubuntu 20.04

**Comment by bkw777** [comment]
I just discovered that I get the same failure if I use make -j 8 (or any number) even with compat still set to 9.

**Comment by bkw777** [comment]
... and the manual for debhelper does say that it does do parallel builds by default starting with version 10
https://www.man7.org/linux/man-pages/man7/debhelper.7.html  
```
       --parallel, --no-parallel
           Control whether parallel builds should be used if underlying
           build system supports them.  The number of parallel jobs is
           controlled by the DEB_BUILD_OPTIONS environment variable
           ("Debian Policy, section 4.9.1") at build time. It might also
           be subject to a build system specific limit.

           If neither option is specified, debhelper currently defaults
           to --parallel in compat 10 (or later) and --no-parallel
           otherwise.

           As an optimization, dh will try to avoid passing these
           options to subprocesses, if they are unnecessary and the only
           options passed.  Notably this happens when DEB_BUILD_OPTIONS
           does not have a parallel parameter (or its value is 1).
```

**Comment by bkw777** [comment]
should be fixed by 980ab380
I put debian/compat = 13 rather than 10, so I don't know if that will cause a problem building on older targets, but I guess we'll see.
